### PR TITLE
Development install script enhanced

### PR DESCRIPTION
**Pull request by SpaSmALX**
Enhanced install script to include options to install frontend and backend separately. Script always does fresh install for both.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -9,16 +56,22 @@ elif [[ ${OS} == 'Darwin' ]]; then
...
+if [ -n "${FRONTEND}" ]; then
+   install_frontend
+fi
 
-# Installs npm node_modules
-pushd deerhunt
-npm install
-popd
+printf "\n\n\nInstallation finished. Don't forget to activate venv by doing:\n"
+printf "   $ source venv/bin/activate\n\n"
```
Comment (KyrelJerome) [inline install.sh:77]: Newline here, usually we let it slide but this is bash so i think it's useful to follow bash convention.

File: install.sh [hunk 2]
```diff
@@ -9,16 +56,22 @@ elif [[ ${OS} == 'Darwin' ]]; then
    brew install npm
 else
     echo 'Windows currently not supported for development'
+    exit;
+fi
+
+if [ -z "${FRONTEND}" ] && [ -z "${BACKEND}" ]; then
```
Comment (KyrelJerome) [inline install.sh:62]: I love this.

File: install.sh (hunk 1 above)
Comment (KyrelJerome) [inline install.sh:77]: Newline here, usually we let it slide but this is bash so i think it's useful to follow bash convention.

File: install.sh [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+   # Deletes existing venv if exists and makes new one.
+   [ -d "venv" ] && rm -rf venv
+   python3 -m venv venv
+   source venv/bin/activate
+
+   # Installs all required backend python modules to run server.
+   pip3 install --upgrade -r ./server/requirements.txt
+}
+
+
```
Comment (KyrelJerome) [inline install.sh:32]: Can we comment that this is parsing options?

**Review by KyrelJerome — CHANGES_REQUESTED** [review]
Looks good to me! This is fire.

File: install.sh (hunk 1 above)
Comment (SpaSmALX) [reply install.sh:77]: Ahh righhht I always forget loool.

**Comment by samnooby** [comment]
alt text

**Pull request merged**